int8 id
float32 velocity_x
float32 velocity_y
float32 velocity_z
float32 velocity_xyz